src/App.js

```diff
@@ -1,12 +1,56 @@
-import React from 'react';
+import React, {Component} from 'react';
 import './App.css';
+import Modal from './components/PokemonChoice';
+//import ChooseCard from './components/ShowChooseCard';
+//import ShowChooseCard from './components/ShowChooseCard';
 
-function App() {
+
+/*function App() {
   return (
-    <div className="App">
-      
-    </div>
-  );
+    
+  )
+}*/
+
+class App extends Component {
+
+  constructor() {
+      super();
```

Comment:
Always write :

```js
constructor(props) {
    super(props);
}
```

Even if it is the App component and it should not receive props. Otherwise sometimes you will forget and you will encounter weird issues you won't understand.